### ISSUE TITLE
gradient가 nan으로 가는 문제

### ISSUE THREAD

**Issue by jazzsaxmafia** [jazzsaxmafia/show_attend_and_tell#1]
cost (loss) 는 정상적으로 나오데 gradient가 nan이 되는 것이 이상함
- Crossentropy loss는  해당 단어에 대한 확률에만 log를 취해 준 것이므로 costs가 정상인 것은 이상하지 않을 수 있음. 해당 단어 말고 나머지 단어들의 확률을 출력해봐야 할 듯.

**Comment by jazzsaxmafia** [comment]
확률 출력해봐도 nan 값은 없음. forward propagation때는 아무 문제 없고 backprop에서 문제 생기는 것 같은데, grad를 출력해봐도 'decode_word_W', 'decode_word_b','decode_lstm_b' 여기까지는 문제가 없고 decode_lstm_W 에서 갑자기 nan이 발견됨.
